packages/beacon-node/test/unit/chain/validation/attestation/validateGossipAttestationsSameAttData.test.ts

```diff
@@ -0,0 +1,128 @@
+import bls from "@chainsafe/bls";
```

Comment:
Spliced top level `describe` file from `attestation.test.ts` as the nature of tests requires different mocks.